webpack.config.js

```diff
@@ -169,6 +169,9 @@ export default {
     ],
   },
   plugins: [
+    new webpack.ProvidePlugin({ // for htmx extensions
+      htmx: 'htmx.org',
+    }),
```

Comment:
What issue is this solving?

Reply:
Ah I see, the htmx exension relies on the global. Not ideal, but I guess acceptable without moving the extension init into our code.

https://github.com/bigskysoftware/idiomorph/blob/8e40c42cc573609eb6863e72fa3403574974dd7d/src/idiomorph-htmx.js#L12

Reply:
Opened https://github.com/bigskysoftware/idiomorph/issues/38 about it.